torch/csrc/jit/register_distributed_ops.cpp

```diff
@@ -59,19 +67,94 @@ RegisterOperators reg_rpc_ops({
           return 0;
         },
         aliasAnalysisFromSchema()),
-});
+    Operator(
+         prim::rpc_async,
+         [](const Node* node) -> Operation {
+           int num_inputs = node->inputs().size();
+           return [num_inputs](Stack& stack) {
+             // Pop inputs from the stack.
+             auto stackIter = stack.end() - num_inputs;
+             auto& dstWorkerNameIValue = *stackIter++;
+             auto& qualifiedNameIValue = *stackIter++;
+             auto& argsTupleIValue = *stackIter++;
+             auto& kwargsDictIValue = *stackIter++;
+             TORCH_INTERNAL_ASSERT(dstWorkerNameIValue.isString());
+             TORCH_INTERNAL_ASSERT(qualifiedNameIValue.isString());
+             TORCH_INTERNAL_ASSERT(argsTupleIValue.isTuple());
+             TORCH_INTERNAL_ASSERT(kwargsDictIValue.isGenericDict());
+
+             // Get FunctionSchema for qualifiedName.
+             auto qualifiedName = c10::QualifiedName(qualifiedNameIValue.toStringRef());
+             std::shared_ptr<script::CompilationUnit> cuPtr;
+             {
+               py::gil_scoped_acquire acquire;
+               cuPtr = get_python_cu();
+             }
+             auto& functionSchema = cuPtr->get_function(qualifiedName).getSchema();
```

Comment:
in the follow up diff, we should move this logic back to rpcTorchscript, so that we do not need to import python cu every call

Reply:
Will do.